### ISSUE TITLE
Bug Mixin function must check for nil Paths

### ISSUE THREAD

**Issue by msample** [go-openapi/analysis#12]
Running the Mixin function will panic if given specs that have no paths. PR to come.

**Comment by AlekSi** [comment]
Is it fixed? I use commit 0473cb67199f68b8b7d90e641afd9e79ad36b851 and it still crashes for me. Example: https://gist.github.com/AlekSi/da4805f35a875f8ee7b9f41f1fd8febc
Trace:
```
2017/08/10 10:01:09 args[0] = 1.json
2017/08/10 10:01:09 args[1:] = [2.json]
panic: assignment to entry in nil map

goroutine 1 [running]:
<CUT>/vendor/github.com/go-openapi/analysis.Mixin(0xc4207bd200, 0xc4205e72f0, 0x1, 0x1, 0x1, 0xc4205e72f0, 0x0)
	/<CUT>/vendor/github.com/go-openapi/analysis/mixin.go:66 +0x1286
<CUT>/vendor/github.com/go-swagger/go-swagger/cmd/swagger/commands.MixinFiles(0x7fff5fbff976, 0x6, 0xc4208224f0, 0x1, 0x2, 0x1b74400, 0xc420082008, 0x16f9560, 0x16e08a8, 0xc4200001a0, ...)
	/<CUT>/vendor/github.com/go-swagger/go-swagger/cmd/swagger/commands/mixin.go:90 +0x224
<CUT>/vendor/github.com/go-swagger/go-swagger/cmd/swagger/commands.(*MixinSpec).Execute(0xc420423840, 0xc4208224e0, 0x2, 0x3, 0xc420423840, 0x1)
	/<CUT>/vendor/github.com/go-swagger/go-swagger/cmd/swagger/commands/mixin.go:46 +0x20c
<CUT>/vendor/github.com/jessevdk/go-flags.(*Parser).ParseArgs(0xc420808240, 0xc420076150, 0x3, 0x3, 0x1010f38, 0x30, 0xc42066a6c0, 0xc420330a00, 0xc4203304c8)
	/<CUT>/vendor/github.com/jessevdk/go-flags/parser.go:316 +0x893
<CUT>/vendor/github.com/jessevdk/go-flags.(*Parser).Parse(0xc420808240, 0x6, 0x17a95b9, 0x6, 0x0, 0x17f18db)
	/<CUT>/vendor/github.com/jessevdk/go-flags/parser.go:186 +0x73
main.main()
	/<CUT>/vendor/github.com/go-swagger/go-swagger/cmd/swagger/swagger.go:105 +0x858
```

**Comment by msample** [comment]
I'll take a look at your example in the next couple days. If it's crashing it shouldn't be to hard to fix.
